service.py

```diff
@@ -205,7 +223,10 @@ def getallsubs(url, allowed_languages, filename="", episode=""):
 
     codes = get_language_codes(allowed_languages)
     if len(codes) < 1:
-        xbmc.executebuiltin((u'Notification(%s,%s)' % (__scriptname__, __language__(32004))).encode('utf-8'))
+        if sys.version_info.major == 3:
+            xbmc.executebuiltin('Notification(%s,%s)' % (__scriptname__, __language__(32004)))
+        else:
+            xbmc.executebuiltin((u'Notification(%s,%s)' % (__scriptname__, __language__(32004))).encode('utf-8'))
```

Comment:
Doesn't this line also work with Python 3 to have as less if statements as possible between Python 3 and 2?

Reply:
Kodi 19 needs a string for executebuiltin and not a byte object which is the result of the encode.